src/DevChatter.Bot.Infra.Ef/Migrations/20180512180351_CreateQuizQuestionsTable.Designer.cs

```diff
@@ -0,0 +1,203 @@
+﻿// <auto-generated />
+using DevChatter.Bot.Core.Data.Model;
```

Comment:
Should this be committed?

Reply:
Yep. It's required in order to perform the database migration.